packages/neuron-wallet/src/controllers/export-debug.ts

```diff
@@ -88,6 +89,32 @@ export default class ExportDebugController {
     })
   }
 
+  private addBundledCKBLog = () => {
+    const name = 'bundled-ckb-log.json'
+    const SIZE_TO_READ = 32_000
+
+    return new Promise((resolve, reject) => {
+      const logPath = path.resolve(ckbPath(), 'data', 'logs', 'run.log')
+      if (!fs.existsSync(logPath)) { reject(new Error("File not found")) }
+
+      const fileStats = fs.statSync(logPath)
+      const position = fileStats.size - SIZE_TO_READ
+
+      fs.open(logPath, 'r', (openErr, fd) => {
```

Comment:
Does it need to call `fs.close` once operations on `fd` is finished at the end?
https://stackoverflow.com/questions/21176733/what-the-scenario-call-fs-close-is-necessary

Reply:
Fixed in https://github.com/nervosnetwork/neuron/pull/1633/commits/5be3afc9f76e6174e6a1f55deae1940283ca1c16